build.gradle.kts

```diff
@@ -233,6 +233,9 @@ allprojects {
     repositories {
         // RAT and Autostyle dependencies
         mavenCentral()
+        flatDir {
+            dirs = setOf(file("libs"))
+        }
```

Comment:
I guess it is here by mistake. Please remove once you no longer need it.

Reply:
It's not a mistake, but this PR is still a draft. Currently the Arrow JARs which are being distributed won't work with this branch so we decided to download some custom JARs and just dump them in for now. This will be resolved once the Arrow project starts distributing JARs which work cross-platform with Gandiva.